### PR TITLE
Add Kirby Battle Royale

### PR DESCRIPTION
**Pull request by robzombie91**
Adds Kirby Battle Royale.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by robzombie91** [comment]
citra-qt_2018-01-28_15-47-23

**Comment by robzombie91** [comment]
citra-qt_2018-01-28_15-48-19

File: games/kirby-battle-royale/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Kirby Battle Royale"
+description = "Kirby Battle Royale is a brawler video game part of the Kirby series. Developed by HAL Laboratory and published by Nintendo, the game was released on the Nintendo 3DS handheld game console in Europe and Japan in November 2017, and in North America in January 2018."
```
Comment (Leo626) [inline games/kirby-battle-royale/game.dat]: The release dates are redundant. So basically everything after "published by Nintendo" you can remove.

**Comment by robzombie91** [comment]
its already reported in the issue tracker as an f-audio pipe 3 missing issue

**Comment by robzombie91** [comment]
https://github.com/citra-emu/citra/issues/3233

**Comment by Leo626** [comment]
Add the issue in game.dat. It should be between `description` and `needs_system_files`. See the example in the [contributing](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md#metadata) guide.

File: games/kirby-battle-royale/game.dat [hunk 2]
```diff
@@ -0,0 +1,9 @@
+title = "Kirby Battle Royale"
+description = "Kirby Battle Royale is a brawler video game part of the Kirby series. Developed by HAL Laboratory and published by Nintendo."
```
Comment (Leo626) [inline games/kirby-battle-royale/game.dat]: Starting the second sentence as "It was developed by" would flow better.

File: games/kirby-battle-royale/game.dat [hunk 3]
```diff
@@ -0,0 +1,10 @@
+title = "Kirby Battle Royale"
+description = "Kirby Battle Royale is a brawler video game part of the Kirby series. It was developed by HAL Laboratory and published by Nintendo."
+github_issue = "F:Audio-Pipe-3 not implemented in current Citra build which affects Background Music - see Citra issue #3233"
```
Comment (Leo626) [inline games/kirby-battle-royale/game.dat]: Github issue should be like this:

```
description = "The Legend of Zelda: Majora's Mask 3D is an action-adventure video game co-developed by Grezzo and Nintendo for the Nintendo 3DS handheld game console. The game is an enhanced remake of The Legend of Zelda: Majora's Mask, which was originally released for the Nintendo 64 home console in 2000. The game was released worldwide in February 2015"
github_issues = [2517]
needs_system_files = false
```

File: games/kirby-battle-royale/game.dat [hunk 4]
```diff
@@ -0,0 +1,10 @@
+title = "Kirby Battle Royale"
+description = "Kirby Battle Royale is a brawler video game part of the Kirby series. It was developed by HAL Laboratory and published by Nintendo."
+github_issues = [3233]
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
```
Comment (CodingKoopa) [inline games/kirby-battle-royale/game.dat]: There should be a USA release.

Comment (robzombie91) [reply games/kirby-battle-royale/game.dat]: There is but i couldn't find the USA title id on 3ds dumps

**Pull request merged**